schema_based_on_diagram.sql

```diff
@@ -0,0 +1,61 @@
+/* Database schema to keep the structure of entire database. */
+
+/* Create table medical_histories*/
+CREATE TABLE medical_histories (
+    id serial primary key,
+    admitted_at timestamp,
+    patient_id int,
+    status varchar(100),
+    FOREIGN KEY (patient_id) REFERENCES patients (id)
+);
+
+CREATE INDEX idx_patient_id ON medical_histories (patient_id);
+
+/* Create table patients*/
+CREATE TABLE patients(
+    id serial primary key,
+    name varchar(100),
+    date_of_birth date
+);
+
+CREATE INDEX index_patients ON patients (id);
+
+
+/* Create table treatments*/
+CREATE TABLE treatments(
+    id serial primary key,
+    type varchar(100),
+    name varchar(100)
+);
+/* establish the many-to-many relationship between medical_histories and treatments*/
+
+CREATE TABLE medical_history_treatments (
+    id serial primary key,
+    medical_history_id int,
+    treatment_id int,
+    FOREIGN KEY (medical_history_id) REFERENCES medical_histories (id),
+    FOREIGN KEY (treatment_id) REFERENCES treatments (id)
```

Comment:
- [ ] Please create indexes for the following foreign keys, as per the requirements.

Reply:
Suggested changes by the code reviewer has been implemented. Screen shot attached below for your reference.
![image](https://github.com/baqar-abbas/clinicDb/assets/117091923/b30386d7-93ad-4c32-b035-3bb20864ec95)